core/lib/config/src/configs/consensus.rs

```diff
@@ -115,6 +115,7 @@ impl RpcConfig {
 /// Config (shared between main node and external node).
 #[derive(Clone, Debug, PartialEq)]
 pub struct ConsensusConfig {
+    pub port: u16,
```

Comment:
what is this port? Is it related to server_addr/public_addr? Why would you want to duplicate this information? Note that there is no requirement for the server_addr port and the public_addr port to be the same.

Reply:
The information is duplicated in some other configs:
https://github.com/matter-labs/zksync-era/blob/main/etc/env/file_based/general.yaml#L30-L31
https://github.com/matter-labs/zksync-era/blob/main/etc/env/file_based/general.yaml#L32-L33
https://github.com/matter-labs/zksync-era/blob/main/etc/env/file_based/general.yaml#L84-L85

We want the configs to be consistent so we can automate the task to allocate ports and avoid overlaps when creating new chains/external node
If at some point we add a new port/url/address, this port allocation will continue working and nothing will break